tools/src/main/scala-2.12/caliban/tools/DescriptorUtils.scala

```diff
@@ -0,0 +1,9 @@
+package caliban.tools
```

Comment:
Since the code is the same between 2.12 and 2.13, I think you can place it into a `scala-2` folder instead of duplicating it.

Reply:
Done.